### DISCUSSION TITLE
X96X4 100mbit RAM4G 32G  - No LAN

### DISCUSSION THREAD

**Discussion by d5stick** [devmfc/debian-on-amlogic#83] [General]
Hi, 

X96X4 100mbit RAM4G 32G  - No LAN
https://nl.aliexpress.com/item/1005005202802923.html?

After some testing i hoped you could help me further. 

My box boots with your image with 2 DTBs, the generic one and the X96x4 but no LAN, WIFI or BT.
The HDMI resolution/location on the screen is a bit off, but that is not an issue.  
The front LCD part works, and shows me the temperature etc. (Nice feature btw)

During the boot i see a lot of errors regarding the fact that is not able to access the Ethernet controller. 

The box itself K255B-SR chip for bluetooth/wifi. The only chip i see near the ethernet is b1601s
Board number BM825_141 V1.3 21272

Any idea if the DTB can be updated/modified or what info do you from me so we could troubleshoot?

**Comment by devmfc** [comment]
I'm sorry, you are out of luck...  100 MB/s ethernet does not work with the current image for the S905X4 (some clock is missing). And the wifi  module (modules is based on Amlogic W150S1) is also an unsupported one at this moment.
